composites/Plugin/SnippetEditor/tests/ReplacementVariableEditorTest.js

```diff
@@ -88,7 +88,7 @@ describe( "suggestionsFilter", () => {
 	} );
 
 	it( "Replacement variables are correctly mapped to suggestions by mapReplacementVariablesToSuggestions.", () => {
-		const expected = [
+		expected = [
```

Comment:
That is not possible, because they `beforeEach` will reset the value to the default.
Thus it can be overwritten in specific tests, but it does not have to be defined for each test separately.